fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingBase.java

```diff
@@ -70,20 +72,39 @@
     private static final Object INDEX_BUILD_LOCK_KEY = 0L;
     private static final Object INDEX_BUILD_SCANNED_RECORDS = 1L;
     private static final Object INDEX_BUILD_TYPE_VERSION = 2L;
+    private static final Object INDEX_BUILD_SCRUB_INDEX_RANGE = 3L;
+    private static final Object INDEX_BUILD_SCRUB_RECORDS_RANGE = 4L;
 
     @Nonnull
     private static final Logger LOGGER = LoggerFactory.getLogger(IndexingBase.class);
     @Nonnull
     protected final IndexingCommon common; // to be used by extenders
     @Nonnull
+    protected final OnlineIndexer.IndexingPolicy policy;
+    @Nullable
+    protected final OnlineIndexer.ScrubbingPolicy scrubbingPolicy;
+    @Nonnull
     private final IndexingThrottle throttle;
 
     private long timeOfLastProgressLogMillis = 0;
     private boolean forceStampOverwrite = false;
 
-    IndexingBase(@Nonnull IndexingCommon common) {
+    IndexingBase(@Nonnull IndexingCommon common,
+                 @Nonnull OnlineIndexer.IndexingPolicy policy) {
+        this.common = common;
+        this.policy = policy;
```

Comment:
No particular reason. I can move it to common.

Reply:
No reason but ease of read. Would you find it more appropriate in common?

Reply:
Moved `scrubbingPolicy` to common. 
policy is a little tricky, as it can be modified changed during fallback (that's why it was not in common in the first place). I left it as is.